app/controllers/dashboard/cards_controller.rb

```diff
@@ -43,6 +45,25 @@ def get_flickr_images
                                                  'url_sq, url_m', 10, 1)
   end
 
+  def parsing_html
+    @parse = CardsService.parsing_html(current_user.id,
+                                      parsing_html_params[:block_id],
+                                      parsing_html_params[:url],
+                                      parsing_html_params[:css_original],
+                                      parsing_html_params[:css_translated])
+
+    if @parse
+      flash.now[:notice] = 'Задача на парсинг сайта успешно поставлена в очередь.'
+    else
+      flash.now[:alert] = 'Проверьте правильность введенных данных.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.